sip editor committee/editors_list.md

```diff
@@ -0,0 +1,127 @@
+
+**Mistiff**: Mistiff has extensive experience managing online communities and a track record of success in creating and fostering them. With immense experience in social media and online marketing, he is well-versed in what it takes to attract and keep a wide variety of consumers in the web3 space. Mistiff is skilled in creating and putting into action, plans for online communities that promote a sense of belonging. He is adept at facilitating talks, handling disputes, and encouraging productive interactions. Mistiff is skilled at using data and analytics to monitor a community's development and progress and to spot areas for improvement. Mistiff is an outstanding communicator and problem-solver in addition to his knowledge of community administration. Mistiff is currently managing multiple web3 communities inside and outside the Stacks ecosystem. He is also a Stacks advocate. Mistiff joined the Stacks ecosystem back in June 2022.
+
+**Rafael Cárdenas**: Rafael is a Staff Software Engineer at Hiro, focused on building and maintaining Hiro’s API infrastructure which includes the Stacks Blockchain API, Ordinals API, Token Metadata API and others, all of which power dozens of apps and wallets built on Stacks and Bitcoin. Rafael has also been an active member and contributor of the Stacks ecosystem since 2021, involved with the development of the core blockchain through the SIP process and participating as a reviewer at the Stacks Accelerator.
+
+**Sleepi**: I am a study coordinator in the medical/pharmacy research with a strong passion for cryptocurrency, having become fascinated with its dynamic nature through day trading. As a community representative, I am committed to educating and supporting others and ensuring fair representation for all members. While valuing privacy, I continue to expand my knowledge in this rapidly-evolving industry and am open to connecting with others to discuss all things crypto. I am an active moderator and support in the official Stacks discord server.
+
+**Tigs / Oli**: tigs / Oli: Yo! The name is tigs and have been a multiplayer pro gamer since the early 90s. Not quite so pro anymore but through this exposure I taught myself Cisco (CCNA) and server administration at a young age. I haven't passed high school (got a GED), have attended college for gaming dev but never completed and have been involved in a few jobs in regards to teaching, inspiring and getting aspiring tech people into the industry as whole. Call me a Jack of all trades. Master of none! I am deeply involved with Horticulture as well (green fingers). I run several bitcoin+lightning nodes, a few miners and am deeply interested in Bitcoin as a saver of humanity, not purely as an investment vehicle. I am also involved with TSOBTC (The School of BTC).
```

Comment:
Remove duplicate text

```suggestion
**Tigs / Oli**: Yo! The name is tigs and have been a multiplayer pro gamer since the early 90s. Not quite so pro anymore but through this exposure I taught myself Cisco (CCNA) and server administration at a young age. I haven't passed high school (got a GED), have attended college for gaming dev but never completed and have been involved in a few jobs in regards to teaching, inspiring and getting aspiring tech people into the industry as whole. Call me a Jack of all trades. Master of none! I am deeply involved with Horticulture as well (green fingers). I run several bitcoin+lightning nodes, a few miners and am deeply interested in Bitcoin as a saver of humanity, not purely as an investment vehicle. I am also involved with TSOBTC (The School of BTC).
```